content/en/news/releases/1.18.x/announcing-1.18/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Announcing Istio 1.18.0
+linktitle: 1.18.0
+subtitle: Major Release
+description: Istio 1.18 Release Announcement.
+publishdate: 2023-06-02
+release: 1.18.00
+aliases:
+    - /news/announcing-1.18.0
+---
+
+We are pleased to announce the release of Istio 1.18. This is the second Istio release of 2023, and the first to ship with Ambient mode! We would like to thank the entire Istio community for helping get the 1.18.0 release published. We would like to thank the Release Managers for this release, `Paul Merrison` from Tetrate, `Kalya Subramanian` from Microsoft and `Xiaopeng Han` from DaoCloud. The release managers would specially like to thank the Test & Release WG lead Eric Van Norman (IBM) for his help and guidance throughout the release cycle. We would also like to thank the maintainers of the Istio work groups and the broader Istio community for helping us throughout the release process with timely feedback, reviews, community testing and for all your support to help ensure a timely release.
+
+{{< relnote >}}
+
+{{< tip >}}
+Istio 1.18.0 is officially supported on Kubernetes versions `1.24` to `1.27`.
+{{< /tip >}}
+
+## What's new
+
+### Ambient Mesh
+Istio 1.18 marks the first release of ambient mesh, a new Istio data plane mode that’s designed for simplified operations, broader application compatibility, and reduced infrastructure cost. For more details see the [announcement blog](https://istio.io/latest/blog/2022/introducing-ambient-mesh/).
+
+### Gateway API Support Improvements
+Istio 1.18 improves Istio's gateway-API support, including support for extra v1beta1 resources and enhancements to automated deployment logic to no longer rely on pod injection.  Users of Gateway API on Istio should review this release's upgrade notes for important guidance on upgrading.
```

Comment:
```suggestion
Istio 1.18 improves Istio's Gateway-API support, including support for extra v1beta1 resources and enhancements to automated deployment logic to no longer rely on pod injection.  Users of Gateway API on Istio should review this release's upgrade notes for important guidance on upgrading.
```
remove the `-` as well?

Reply:
I think this should be something like `Istio 1.18 improves support for the Kubernetes Gateway API.`? It might be misinterpreted as part of the Istio's API.